ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml

```diff
@@ -314,7 +314,7 @@ tests:
     - ref: fips-check
     workflow: openshift-e2e-gcp
 - as: console-aws
-  interval: 48h
+  interval: 24h
```

Comment:
We'll probably need to scale this back based on https://issues.redhat.com/browse/DPTP-3048

Reply:
@spadgett Updated this from twice a month to every day as we have still a lot of backports for 4.9. See my updated initial comment for all changes.